runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -499,45 +493,34 @@ protected DataflowRunner(DataflowPipelineOptions options) {
           PTransformOverride.of(
               PTransformMatchers.writeWithRunnerDeterminedSharding(),
               new StreamingShardedWriteFactory(options)));
-      if (fnApiEnabled) {
-        overridesBuilder.add(
-            PTransformOverride.of(
-                PTransformMatchers.classEqualTo(Create.Values.class),
-                new StreamingFnApiCreateOverrideFactory()));
-      }
 
       overridesBuilder.add(
           PTransformOverride.of(
               PTransformMatchers.groupWithShardableStates(),
               new GroupIntoBatchesOverride.StreamingGroupIntoBatchesWithShardedKeyOverrideFactory(
                   this)));
 
-      if (!fnApiEnabled) {
-        overridesBuilder
-            .add(
-                // Streaming Bounded Read is implemented in terms of Streaming Unbounded Read, and
-                // must precede it
-                PTransformOverride.of(
-                    PTransformMatchers.classEqualTo(Read.Bounded.class),
-                    new StreamingBoundedReadOverrideFactory()))
-            .add(
-                PTransformOverride.of(
-                    PTransformMatchers.classEqualTo(Read.Unbounded.class),
-                    new StreamingUnboundedReadOverrideFactory()));
-      }
+      overridesBuilder
+          .add(
+              // Streaming Bounded Read is implemented in terms of Streaming Unbounded Read, and
+              // must precede it
+              PTransformOverride.of(
+                  PTransformMatchers.classEqualTo(Read.Bounded.class),
+                  new StreamingBoundedReadOverrideFactory()))
+          .add(
+              PTransformOverride.of(
+                  PTransformMatchers.classEqualTo(Read.Unbounded.class),
+                  new StreamingUnboundedReadOverrideFactory()));
+
+      overridesBuilder.add(
+          PTransformOverride.of(
+              PTransformMatchers.classEqualTo(View.CreatePCollectionView.class),
+              new StreamingCreatePCollectionViewFactory()));
 
-      if (!fnApiEnabled) {
-        overridesBuilder.add(
-            PTransformOverride.of(
-                PTransformMatchers.classEqualTo(View.CreatePCollectionView.class),
-                new StreamingCreatePCollectionViewFactory()));
-      }
       // Dataflow Streaming runner overrides the SPLITTABLE_PROCESS_KEYED transform
       // natively in the Dataflow service.
     } else {
-      if (!fnApiEnabled) {
-        overridesBuilder.add(SplittableParDo.PRIMITIVE_BOUNDED_READ_OVERRIDE);
-      }
+      overridesBuilder.add(SplittableParDo.PRIMITIVE_BOUNDED_READ_OVERRIDE);
```

Comment:
I just noticed that we only swap SDF wrapper out when in batch and we are using SDF wrapper in streaming on runner_v1 now. Though runner_v1 supports Splittable DoFn, should we swap the read for streaming as well?